Exercise_D and E/assets/style/index.css

```diff
@@ -2,3 +2,82 @@
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+* {
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins', sans-serif;
+}
+
+.header {
+    min-height: 100vh;
+    width: 100%;
+    background-image: linear-gradient(rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7)), url(../images/flowershop.jpg);
+    background-position: center;
+    background-size: cover;
+    position: relative;
+}
+
+nav {
+    background-color: white;
+}
+
+.text-box {
```

Comment:
If you target the container that encapsulates this class and apply flex, then you can easily put it into the middle of any page